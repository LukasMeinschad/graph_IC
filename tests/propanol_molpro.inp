memory, 500, m ! Memory Requirements                                                                                                                                                     
gthresh, optgrad=1.d-7,twoint=1.d-14,prefac=1.d-16 ! Guntram Rezept um numerischen Schmutz zu entfernen
orient,mass
geometry={
C1,         -3.59438,        1.26763,       -0.04227
C2,         -2.06872,        1.25426,       -0.03552
H1,         -3.98151,        1.62418,        0.93606
H2,         -3.97482,        0.24008,       -0.22216
H3,         -3.97123,        1.93265,       -0.84842
H4,         -1.71642,        0.56796,        0.76479
C3,         -1.50339,        2.65474,        0.20980
H5,         -1.70438,        0.87618,       -1.01534
H6,         -1.86455,        3.06534,        1.18074
H7,         -1.84981,        3.34467,       -0.58914
 O,         -0.10245,        2.62381,        0.18791
H8,          0.18832,        2.26681,        1.06760
}
   
mass, iso
basis=6-311g(d,p)
 
{hf
start,atden}
 
optg
{frequencies, symm=auto, print=0,analytical}
put,molden,propanol.molden

