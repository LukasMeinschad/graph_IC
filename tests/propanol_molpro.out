
 Working directory              : /tmp/lme/molpro.q4BlNhR43n/
 Global scratch directory       : /tmp/lme/molpro.q4BlNhR43n/
 Wavefunction directory         : /home/lme/wfu/
 Main file repository           : /tmp/lme/molpro.q4BlNhR43n/

 id        : liedlgroup

 Nodes     nprocs
 sv1-c724     6
 GA implementation: MPI file
 GA implementation (serial work in mppx): MPI file

 Using customized tuning parameters: mindgm=1; mindgv=20; mindgc=4; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=sf  


 Variables initialized (1045), CPU time= 0.00 sec
 memory, 500, m                                                                  ! Memory Requirements
 gthresh, optgrad=1.d-7,twoint=1.d-14,prefac=1.d-16                            ! Guntram Rezept um numerischen Schmutz zu entfernen
 orient,mass
 geometry={
 C1,         -3.59438,        1.26763,       -0.04227
 C2,         -2.06872,        1.25426,       -0.03552
 H1,         -3.98151,        1.62418,        0.93606
 H2,         -3.97482,        0.24008,       -0.22216
 H3,         -3.97123,        1.93265,       -0.84842
 H4,         -1.71642,        0.56796,        0.76479
 C3,         -1.50339,        2.65474,        0.20980
 H5,         -1.70438,        0.87618,       -1.01534
 H6,         -1.86455,        3.06534,        1.18074
 H7,         -1.84981,        3.34467,       -0.58914
  O,         -0.10245,        2.62381,        0.18791
 H8,          0.18832,        2.26681,        1.06760
 }
 
 mass, iso
 basis=6-311g(d,p)
 
 {hf
 start,atden}
 
 optg
 {frequencies, symm=auto, print=0,analytical}
 put,molden,propanol.molden
 
 Commands initialized (836), CPU time= 0.00 sec, 720 directives.
 Default parameters read. Elapsed time= 0.11 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2024.2 linked Tue Oct  8 23:14:51 2024


 **********************************************************************************************************************************
 LABEL *                                                                                                                                                                 
  (6 PROC) 64 bit mpp version                                                            DATE: 09-Jan-25          TIME: 11:50:19  
 **********************************************************************************************************************************

 SHA1:             5a9813c25d9a7b7fe243115095799e27f07d421d
 **********************************************************************************************************************************

 Memory per process:       500 MW
 Total memory per node:   3000 MW

 GA preallocation disabled
 GA check disabled

 Variable memory set to 500.0 MW


 THRESHOLDS:

 ZERO    =  1.00D-12  ONEINT  =  1.00D-12  TWOINT  =  1.00D-14  PREFAC  =  1.00D-16  XXXXXX  =  1.00D-09  EORDER  =  1.00D-04
 ENERGY  =  0.00D+00  ETEST   =  0.00D+00  EDENS   =  0.00D+00  THRDEDEF=  1.00D-06  GRADIENT=  1.00D-05  STEP    =  1.00D-03
 ORBITAL =  1.00D-05  CIVEC   =  1.00D-05  COEFF   =  1.00D-04  PRINTCI =  5.00D-02  PUNCHCI =  9.90D+01  OPTGRAD =  1.00D-07
 OPTENERG=  1.00D-06  OPTSTEP =  3.00D-04  THRGRAD =  1.00D-10  COMPRESS=  1.00D-11  VARMIN  =  1.00D-07  VARMAX  =  1.00D-03
 THRDOUB =  0.00D+00  THRDIV  =  1.00D-05  THRRED  =  1.00D-07  THRPSP  =  1.00D+00  THRDC   =  1.00D-10  THRCS   =  1.00D-10
 THRNRM  =  1.00D-08  THREQ   =  0.00D+00  THRDE   =  1.00D+00  THRREF  =  1.00D-05  SPARFAC =  1.00D+00  THRDLP  =  1.00D-07
 THRDIA  =  1.00D-10  THRDLS  =  1.00D-07  THRGPS  =  0.00D+00  THRKEX  =  0.00D+00  THRDIS  =  2.00D-01  THRVAR  =  1.00D-10
 THRLOC  =  1.00D-06  THRGAP  =  1.00D-06  THRLOCT = -1.00D+00  THRGAPT = -1.00D+00  THRORB  =  1.00D-06  THRMLTP =  0.00D+00
 THRCPQCI=  1.00D-10  KEXTA   =  0.00D+00  THRCOARS=  0.00D+00  SYMTOL  =  1.00D-06  GRADTOL =  1.00D-06  THROVL  =  1.00D-08
 THRORTH =  1.00D-08  THRDLOVL= -1.00D+00  GRID    =  1.00D-06  GRIDMAX =  1.00D-03  DTMAX   =  0.00D+00  THROCCDE=  1.00D-01
 THROCCDE=  1.00D-01  THRPRINT=  2.50D-01

 ZCENTRE=MASS


 Geometry recognized as XYZ



 Using masses of most abundant isotopes

 **********************************************************************************************************************************
 SETTING BASIS          =    6-311G(D,P)


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry C      S 6-311G               selected for orbital group  1
 Library entry C      P 6-311G               selected for orbital group  1
 Library entry C      D 6-311G*              selected for orbital group  1
 Library entry H      S 6-311G               selected for orbital group  2
 Library entry H      P 6-311G**             selected for orbital group  2
 Library entry O      S 6-311G               selected for orbital group  5
 Library entry O      P 6-311G               selected for orbital group  5
 Library entry O      D 6-311G*              selected for orbital group  5


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Asymmetric top
 Rotational constants:       3.8441666      3.6091852     25.2799560 GHz

 Point group  C1  



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  C1      6.00    0.268096589    0.028261367   -3.681164366
   2  C2      6.00   -0.995329359   -0.045468957   -1.090552948
   3  H1      1.00    1.393468264    1.787646891   -3.893882846
   4  H2      1.00   -1.198950679   -0.016576838   -5.180737818
   5  H3      1.00    1.529837156   -1.631735760   -3.925363700
   6  H4      1.00   -2.274324534    1.610626108   -0.906172095
   7  C3      6.00    0.966066223    0.013151169    1.033075345
   8  H5      1.00   -2.138892119   -1.800657829   -0.934132687
   9  H6      1.00    2.160191626    1.743021165    0.906599381
  10  H7      1.00    2.246705933   -1.641678736    0.861844412
  11  O       8.00   -0.231578650   -0.102204468    3.392345690
  12  H8      1.00   -0.886464472    1.619712769    3.748242176

 Bond lengths in Bohr (Angstrom)

 1-2  2.883218480  1-3  2.099320838  1-4  2.098322828  1-5  2.099336332  2-6  2.100589399
     ( 1.525733514)     ( 1.110912746)     ( 1.110384622)     ( 1.110920945)     ( 1.111584039)

  2- 7  2.891419388   2- 8  2.100688234   7- 9  2.105797201   7-10  2.099480915   7-11  2.648361139
       ( 1.530073247)       ( 1.111636341)       ( 1.114339889)       ( 1.110997455)       ( 1.401452361)

 11-12  1.876309335
       ( 0.992900140)

 Bond angles

  1-2-6  108.99680329   1-2-7  111.23227972   1-2-8  109.07564374   2-1-3  110.32707582

  2- 1- 4  109.58536722   2- 1- 5  110.34442879   2- 7- 9  110.92904902   2- 7-10  109.74886940

  2- 7-11  110.28187620   3- 1- 4  108.68018600   3- 1- 5  109.19074152   4- 1- 5  108.67362819

  6- 2- 7  109.42537060   6- 2- 8  108.70745514   7- 2- 8  109.36153006   7-11-12  106.66879756

  9- 7-10  107.25957071   9- 7-11  110.22607726  10- 7-11  108.31065811

 NUCLEAR CHARGE:                   34
 NUMBER OF PRIMITIVE AOS:         192
 NUMBER OF SYMMETRY AOS:          188
 NUMBER OF CONTRACTIONS:          120   (  120A   )
 NUMBER OF INNER CORE ORBITALS:     0   (    0A   )
 NUMBER OF OUTER CORE ORBITALS:     4   (    4A   )
 NUMBER OF VALENCE ORBITALS:       24   (   24A   )


 NUCLEAR REPULSION ENERGY  130.32946621


 Eigenvalues of metric

         1 0.238E-02 0.551E-02 0.104E-01 0.125E-01 0.177E-01 0.232E-01 0.239E-01 0.246E-01


 Contracted 2-electron integrals neglected if value below      1.0D-14
 AO integral compression algorithm  1   Integral accuracy      1.0D-14

     158.859 MB (compressed) written to integral file ( 67.0%)

     Node minimum: 23.593 MB, node maximum: 28.836 MB


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:    4395930.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:    4395930      RECORD LENGTH: 524288

 Memory used in sort:       4.95 MW

 SORT1 READ    29624924. AND WROTE     4117845. INTEGRALS IN     12 RECORDS. CPU TIME:     0.41 SEC, REAL TIME:     0.45 SEC
 SORT2 READ    24035056. AND WROTE    26357430. INTEGRALS IN    612 RECORDS. CPU TIME:     0.09 SEC, REAL TIME:     0.11 SEC

 Node minimum:     4389880.  Node maximum:     4395930. integrals

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      18       29.55       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         1.21      1.04
 REAL TIME  *         1.64 SEC
 DISK USED  *        30.08 MB (local),      604.39 MB (total)
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************


 Program * Restricted Hartree-Fock


 Generating basis MINAO
  Atom=C        basis=MINAO
  Atom=H        basis=MINAO
  Atom=O        basis=MINAO
 Number of basis functions:      28

 Orbital guess generated from atomic densities.

 Initial occupancy:  17

 NELEC=   34   SYM=1   MS2= 0   THRE=1.0D-08   THRD=1.0D-06   THRG=1.0D-06  HFMA2=F  DIIS_START=2   DIIS_MAX=10   DIIS_INCORE=F

 Level shifts:    0.00 (CLOSED)    0.00 (OPEN)    0.30 (GAP_MIN)

 ITER           ETOT              DE          GRAD        DDIFF     DIIS  NEXP   TIME(IT)  TIME(TOT)  DIAG
   1     -193.09706489    -193.09706489     0.00D+00     0.37D-01     0     0       0.03      0.09    start
   2     -193.15206644      -0.05500155     0.56D-02     0.44D-02     1     0       0.04      0.13    diag
   3     -193.16140146      -0.00933502     0.27D-02     0.16D-02     2     0       0.03      0.16    diag
   4     -193.16251375      -0.00111229     0.69D-03     0.60D-03     3     0       0.03      0.19    diag
   5     -193.16257716      -0.00006341     0.16D-03     0.16D-03     4     0       0.03      0.22    diag
   6     -193.16258229      -0.00000513     0.45D-04     0.50D-04     5     0       0.03      0.25    diag
   7     -193.16258260      -0.00000031     0.96D-05     0.14D-04     6     0       0.03      0.28    diag
   8     -193.16258262      -0.00000002     0.20D-05     0.34D-05     7     0       0.04      0.32    fixocc
   9     -193.16258262      -0.00000000     0.53D-06     0.73D-06     8     0       0.03      0.35    diag
  10     -193.16258262      -0.00000000     0.16D-06     0.18D-06     0     0       0.03      0.38    diag/orth

 Final occupancy:  17

 !RHF STATE 1.1 Energy               -193.162582620024
  RHF One-electron energy            -520.377186850964
  RHF Two-electron energy             196.885138017210
  RHF Kinetic energy                  192.850471507606
  RHF Nuclear energy                  130.329466213729
  RHF Virial quotient                  -1.001618409901

 !RHF STATE 1.1 Dipole moment           0.01082005     0.57054229    -0.40251619
 Dipole moment /Debye                   0.02750182     1.45017385    -1.02309411

 Orbital energies:

           1.1          2.1          3.1          4.1          5.1          6.1          7.1          8.1          9.1         10.1
    -20.546862   -11.274589   -11.223865   -11.222949    -1.354693    -1.048242    -0.906241    -0.804255    -0.690082    -0.628660

          11.1         12.1         13.1         14.1         15.1         16.1         17.1         18.1         19.1
     -0.596964    -0.585652    -0.533675    -0.498599    -0.494193    -0.466572    -0.427359     0.148704     0.161646


 HOMO     17.1    -0.427359 =     -11.6290eV
 LUMO     18.1     0.148704 =       4.0464eV
 LUMO-HOMO         0.576063 =      15.6755eV

 Orbitals saved in record  2100.2


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      18       29.55       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    OPER   

              2       4        0.79       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL    HF-SCF       INT
 CPU TIMES  *         1.60      0.38      1.04
 REAL TIME  *         2.09 SEC
 DISK USED  *        31.37 MB (local),      612.16 MB (total)
 GA USED    *         0.00 MB       (max)       0.00 MB       (current)
 **********************************************************************************************************************************


 PROGRAM * OPT (Geometry optimization)     Authors: F. Eckert and H.-J. Werner


 BFGS update of hessian

 *** Long output written to logfile /home/lme/graph_IC/tests/propanol_molpro.log ***


 Geometry optimization using default procedure for command HF-SCF

 Geometry written to block  1 of record 700

 PQS Optimization using delocalized internal coordinates

 Point group c1. Number of degrees of freedom:   30

 ITER.   ENERGY(OLD)    ENERGY(NEW)      DE          GRADMAX     GRADNORM    GRADRMS     STEPMAX     STEPLEN     STEPRMS   CPU-time
   1  -193.16258262  -193.16928223    -0.00669961  0.03466769  0.07175576  0.01310075  0.09191645  0.22584531  0.04123352      2.58
   2  -193.16928223  -193.16949874    -0.00021651  0.00615056  0.00927439  0.00169326  0.04376396  0.09269058  0.01692291      4.86
   3  -193.16949874  -193.16958540    -0.00008665  0.00148209  0.00293557  0.00053596  0.04657774  0.08518363  0.01555233      7.18
